Source/DataModels/Resources/ResourceMesh.h

```diff
@@ -146,6 +187,21 @@ inline const std::vector<std::vector<unsigned int> >& ResourceMesh::GetFacesIndi
 	return facesIndices;
 }
 
+inline const std::vector<Bone>& ResourceMesh::GetBones() const 
+{
+	return bones;
+}
+
+inline const std::vector<Attach>& ResourceMesh::GetAttaches() const 
+{
+	return attaches;
+}
+
+inline const std::vector<unsigned int>& ResourceMesh::GetNumWeights() const
+{
+	return this->numWeights;
```

Comment:
can we do it this way? If it works I think is more simple
```suggestion
	return numWeights;
```

Reply:
I think that according to the programming rules it has to be in this format.